llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -28,7 +28,6 @@
  * @brief LLPC source file: contains implementation of class Llpc::SpirvLowerGlobal.
  ***********************************************************************************************************************
  */
-
 #include "llvm/ADT/SmallSet.h"
```

Comment:
I'm not sure whether this particular changes is worth it, but it doesn't really hurt either...

Reply:
I forgot to add that to #468, the include lines start directly after the comments everywhere else.
I could move it to the other patch if wanted.

Reply:
That would be good, yes.